# French translations for PACKAGE package.
# Copyright (C) 2010 THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# Vigilo language team <contact@projet-vigilo.org>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: vigilo.connector-metro\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2010-02-19 09:00+0100\n"
"PO-Revision-Date: 2010-02-19 09:00+0100\n"
"Last-Translator: Automatically generated\n"
"Language-Team: none\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: src/vigilo/connector_metro/nodetorrdtool.py:49
msgid "Got exception"
msgstr "Exception levée"

#: src/vigilo/connector_metro/nodetorrdtool.py:70
msgid "Connection initialized"
msgstr "Connexion initialisée"

#: src/vigilo/connector_metro/nodetorrdtool.py:81
#, python-format
msgid "Unable to create directory '%(dir)s'"
msgstr "Impossible de créer le répertoire '%(dir)s'"

#: src/vigilo/connector_metro/nodetorrdtool.py:85
#, python-format
msgid "Unable to write in the directory '%(dir)s'"
msgstr "Impossible d'écrire dans le répertoire '%(dir)s'"

#: src/vigilo/connector_metro/nodetorrdtool.py:93
#, python-format
msgid "started rrdtool subprocess: pid %(pid)d"
msgstr "Sous-processus rrdtool démarré : pid %(pid)d"

#: src/vigilo/connector_metro/nodetorrdtool.py:98
#, python-format
msgid ""
"Unable to start \"%(rrdtool)s\". Make sure RRDtool is installed and you have "
"permissions to use it."
msgstr "Impossible de démarrer \"%(rrdtool)s\". Assurez-vous que RRDtool est "
"installé et que vous disposez bien des droits nécessaires pour l'utiliser."

#: src/vigilo/connector_metro/nodetorrdtool.py:106
#, python-format
msgid "RRDtool seemed to exit with return code %(returncode)d, restarting it..."
msgstr "RRDtool semble s'être arrêté avec le code de retour %(returncode)d, "
"redémarrage en cours."

#: src/vigilo/connector_metro/nodetorrdtool.py:134
#, python-format
msgid ""
"'RRDtool send back Error message on this command'%(cmd)s' with this file '%"
"(filename)s'. RRDtool replied with: '%(msg)s'"
msgstr "RRDtool a répondu par un message d'erreur à la commande "
"'%(cmd)s' concernant le fichier '%(filename)s' : '%(msg)s'."

#: src/vigilo/connector_metro/nodetorrdtool.py:161
#, python-format
msgid "Unable to create the directory '%(dir)s'"
msgstr "Impossible de créer le répertoire '%(dir)s'"

#: src/vigilo/connector_metro/nodetorrdtool.py:167
#, python-format
msgid ""
"Host with this datasource '%(host_ds)s' not found in the configuration file (%"
"(fileconf)s) !"
msgstr "L'hôte correspondant au datasource '%(host_ds)s' est "
"introuvable dans le fichier de configuration (%(fileconf)s) !"

#: src/vigilo/connector_metro/nodetorrdtool.py:197
#, python-format
msgid "'%(msgtype)s' is not a valid message type for metrology"
msgstr "'%(msgtype)s' n'est pas un message valide pour la metrologie"

#: src/vigilo/connector_metro/nodetorrdtool.py:209
#, python-format
msgid "not a valid perf message (%(i)s is missing '%(perfmsg)s')"
msgstr "Message de performance invalide (%(i)s est manquant '%(perfmsg)s')"

#: src/vigilo/connector_metro/nodetorrdtool.py:224
#, python-format
msgid "Unable to create the directory '%s'"
msgstr "Impossible de créer le répertoire '%s'"

#: src/vigilo/connector_metro/nodetorrdtool.py:247
#, python-format
msgid "Chat message to forward: %s"
msgstr "Message à transmettre : %s"

#: src/vigilo/connector_metro/main.py:35
msgid "Please set the path to the configuration "
"file generated by Vigiconf in the settings.ini."
msgstr "Veuillez spécifier dans le fichier settings.ini le "
"chemin vers le fichier de configuration fourni par Vigiconf."
